README.md

```diff
@@ -1,3 +1,96 @@
 ![.Net](https://github.com/DFE-Digital/meeting-timer/workflows/.NET/badge.svg)
 
-# meeting-timer
+# MeetingTimer
+
+MeetingTimer is a... It uses...
+
+# Getting started
+
+## Prerequisites
+
+- [.NET SDK](https://docs.microsoft.com/en-us/dotnet/core/sdk) - includes the [.NET CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/)
+- [Node.js](https://nodejs.org/en/download/) - includes [npm](https://docs.npmjs.com/)
+
+To confirm that the prerequisites are installed, the following should return a version number:
+
+```js
+dotnet --version
+node --version
+npm --version
+```
+
+## Installation
+
+Clone the project:
+
+```js
+git clone https://github.com/DFE-Digital/meeting-timer.git
+```
+
+### ASP.NET Core
+
+Navigate to `./MeetingTimer`
+
+Restore the .NET project dependencies:
+
+```js
+dotnet restore
+```
+
+### React
+
+Navigate to `./MeetingTimer/ClientApp`
+
+Install Node.js dependencies:
+
+```js
+npm install
+```
+
+## Run the project
+
+1. Update the database. See [Apply Migration](#apply-migration). This will create a SQLite database file at `./MeetingTimer/Data`
+
+2. Run the application. Navigate to `./MeetingTimer`:
+
+```js
+dotnet run
+```
+
+3. The app will be available at:
+
+```js
+https://localhost:5001/
+```
+
+# Swagger UI
+
+[Swashbuckle](https://github.com/domaindrivendev/Swashbuckle.AspNetCore) is used to automatically generate API documentation via Swagger.
+
+To view the interactive documentation with SwaggerUI go to:
+
+```js
+https://localhost:<PORT>/api-documentation
+```
+
+# Migrations
+
+In order to use the Entity Framework core CLI tools they must first be installed. You can find some help [here](https://docs.microsoft.com/en-us/ef/core/cli/dotnet).
+
```

Comment:
Might be worth just putting the command to install the tools `dotnet tool install --global dotnet-ef` in here too

Reply:
🙏 Can you check to update? I have added it to the prerequisites too